settings.gradle

```diff
@@ -192,3 +194,186 @@ if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
   }
 }
 
+dependencyResolutionManagement {
```

Comment:
do you know if dependabot is able to automatically update these dependencies whenever there's a new version? I think it would be good to define the versions in a `versions.toml` file, wdyt?

Reply:
> do you know if dependabot is able to automatically update these dependencies whenever there's a new version? 
can't tell right away, will look into that

>I think it would be good to define the versions in a versions.toml file, wdyt?
I was considering it but decided against that because of flexibility, for example I'm not sure if we can get scalaVersion property in toml file, it's extracted from a system property atm in settings.gradle

Reply:
I found that toml files [are supported](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#gradle) by dependabot but versionCatalog in settings.gradle [is not yet supported](https://github.com/dependabot/dependabot-core/issues/3121#issuecomment-1467922666). 
I wonder if a hybrid approach is possible, I will investigate. By hybrid I mean if we leave only dynamic versions in settings.gradle file and move all the remaining ones to toml file, most would go to toml file as we have just about a dozen of dynamic dependency versions out of couple hundreds in total. Though that even if it works would  complicate project setup and make it more difficult to grasp for a project newcomers.

Reply:
I would aim for putting most things into a version.toml file. Scala versions aren't updated too often, so I think in that case it's fine to define the scala version separately (in a gradle file)

Reply:
I moved all static dependencies to toml file.